src/main/java/frc/robot/PhotonRunnable.java

```diff
@@ -100,6 +102,17 @@ public PhotonRunnable(String[] cameraNames, Transform3d[] robotToCameras, AddVis
       photonPoseEstimators[i] = new PhotonPoseEstimator(
           layout, MULTI_TAG_PNP_ON_COPROCESSOR, new PhotonCamera(cameraNames[i]), robotToCameras[i]);
     }
+    final PhotonCamera leftCamera = new PhotonCamera("Left");
+    final PhotonCamera rightCamera = new PhotonCamera("Right");
+    if (leftCamera.isConnected() && rightCamera.isConnected()) {
+      isCameraConnected = true;
+      leftCamera.close();
+      rightCamera.close();
+    }
```

Comment:
We need to do a little thinking on how to detect if the cameras are connected. What you have would work, but only if the cameras are connected when `PhotonRunnable` is instantiated. It only runs once, so if they come online later it wouldn't detect it. It would also fail to detect if the cameras were connected, but then disconnected later.

We probably need to have a timeout where we consider a camera disconnected if it hasn't sent any data for a while, maybe one second. We have to do something within the `run()` method, this is the thread's code that runs continuously. One consideration is that the thread's code blocks (waits) at `WPIUtilJNI.waitForObjects()` if there is no new camera data, so the thread can tell you when the cameras _are_ connected, but something else will have to watch for when we stop receiving data.